web/src/lib/components/MetaTag.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+    import { onMount } from 'svelte';
+
+    export let title: string = "Plebeian Market";
+    export let description: string = "Self-sovereign marketplace built on Bitcoin and Lightning";
+    export let image: string = "https://plebeian.market/images/logo.jpg";
+    export let domain: string  = "plebeian.market";
+
+    let defaultTitle = "Plebeian Market"
+    let baseTitle: string = " | Plebeian Market"
+    let url: string;
+
+    onMount(() => {
+        url = window.location.href
+        // concat title if not on home page
+        if (title !== defaultTitle) {
```

Comment:
So this components checks the title it received and compares to "a default". This seems quite error prone because it means that every other component needs to be aware of what the default is. Simply pass "null" and let the component decide whether we are in the "default" case?